src/App.jsx

```diff
@@ -1,104 +1,28 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
```

Comment:
update imports order:

example:


```

import { PostList } from './components/PostList/PostList';

import postsFromServer from './api/posts.json';
import commentsFromServer from './api/comments.json';
import usersFromServer from './api/users.json';

import './App.scss';
```

Reply:
Hi! Could you please explain, why I should place { PostList } import first? 
I thought they should be in the order of how I use them in my code 
I agree that its better to divide different types of imports with blank lines (eg components, jsons, styles), but I dont understand why I should change the order 
Thanks!

Reply:
Hello again! 
I added almost all improvements exept destructing, because in this case I find it really hard to read :(
Thank you!

Reply:
Useful info about imports
https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf

Reply:
`exept destructing` you have to do destructuring he according to our style guide:

![Screenshot 2024-04-19 at 15 09 08](https://github.com/mate-academy/react_static-list-of-posts-js/assets/45732323/87eca500-4284-447e-bc23-d68603ddccaa)

Reply:
Please fix this moment and don't forget about this one in the next tasks)))